app.yaml

```diff
@@ -1,18 +1,24 @@
-runtime: python
-env: flex
-entrypoint: gunicorn -b :$PORT main:app
+# Copyright 2018 Google LLC
```

Comment:
I think we don't need these copyright text because we are not employees of Google LLC.